build.gradle.kts

```diff
@@ -98,6 +98,8 @@ val rat by tasks.getting(org.nosphere.apache.rat.RatTask::class) {
     verbose.set(true)
     // Note: patterns are in non-standard syntax for RAT, so we use exclude(..) instead of excludeFile
     exclude(rootDir.resolve(".ratignore").readLines())
+    exclude("src/dist-check/temp")
+    dependsOn(":src:dist:copyBinLibs", ":src:dist:copyLibs")
```

Comment:
I guess this is not intended. `rat` task should verify the license headers for the files present under source control, so there's no sense in `dependsOn(...copyBinLibs)`. Would you clarify what was the warning that required `dependsOn`?

Reply:
This change is intended. It is necessary to support incremental builds for the `rat` task:

```
Execution optimizations have been disabled for task ':rat' to ensure correctness due to the following reasons:
- Gradle detected a problem with the following location: '/Users/sandra/experiments/gradle-enterprise-gradle-build-validation/.data/01-validate-incremental-building/20230105T073724-63b67024/build_jmeter'. Reason: Task ':rat' uses this output of task ':src:dist:copyBinLibs' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed. Please refer to https://docs.gradle.org/7.3/userguide/validation_problems.html#implicit_dependency for more details about this problem.
- Gradle detected a problem with the following location: '/Users/sandra/experiments/gradle-enterprise-gradle-build-validation/.data/01-validate-incremental-building/20230105T073724-63b67024/build_jmeter'. Reason: Task ':rat' uses this output of task ':src:dist:copyLibs' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed. Please refer to https://docs.gradle.org/7.3/userguide/validation_problems.html#implicit_dependency for more details about this problem.
```

Reply:
That is false positive.
`rat` ignores `lib/**/*.jar` while `copyLibs` copies only `*.jar` files.

I'm inlined to add `mustRunAfter` to silence Gradle warning.